integration_tests/mfa-change-demo/README.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+Feature store demo.
+
+We use 'simulators' to simulate data input.
+
+Then all messages will be sent to the 'server' and written in Kafka -> RisingWave. RisingWave will process the data based on pre-defined operations.
+
+We also utilize the 'simulator' to simulate user queries to our 'feature'. The 'server' will receive requests -> query data -> and return results.
+
+If we intend to modify our business logic, we simply need to update the materialized view within our RisingWave by using SQL statements.
+
+## Installation
+
+Run it in local.
+
+1. Build docker. Kafka RisingWave and Feature Store.
+
+```docker compose up --build```
```

Comment:
Can we explain what the demo is? I know it is mentioned in the blog already but I think it is better to make the demo self-explained